config/private.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import "time"
+
+// Private contains configurations which are private, adding it to be part of config without exposing to customers.
+type Private struct {
+	OAuthTokensPurgeInterval int `json:"-"`
```

Comment:
The field will need a comment per our doc conventions.

I'd like to see Config.OriginalPath moved here as it's the single field in Config that belongs here.